code/_onclick/hud/rendering/render_plate.dm

```diff
@@ -80,5 +80,5 @@
 	relay.mouse_opacity = mouse_opacity
 	relay.name = render_target
 	mymob.client.screen += relay
-	if(plane != BLACKNESS_PLANE) //internal snowflake do not touch
+	if(blend_mode != BLEND_MULTIPLY)
```

Comment:
what was the reasoning behind this change?

Reply:
I do not know but it, managed to cause me grief.
I'm gonna bully tivi

Reply:
I think it's because anything that uses non render relays needs to have blend_mode still matter, but I'm still not sure why this check existed in the first place.